target/linux/generic/config-6.6

```diff
@@ -324,7 +324,9 @@ CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MIN=8
 # CONFIG_ARM64_64K_PAGES is not set
 # CONFIG_ARM64_AMU_EXTN is not set
 # CONFIG_ARM64_BTI is not set
+CONFIG_ARM64_CNP=y
```

Comment:
This is supported since ARMv8.2 I think we should enable it even when most SoCs in OpenWrt do not support it yet, but some do.

Reply:
Done